src/main/java/com/algorand/algosdk/transaction/Transaction.java

```diff
@@ -498,21 +524,19 @@ private Transaction(// Asset xfer transaction
      * @param genesisID corresponds to the id of the network
      * @param genesisHash corresponds to the base64-encoded hash of the genesis
      * of the network
-     * @param assetCreator is the address of the asset creator
      * @param assetIndex is the asset index
      **/
-    public static Transaction createAcceptAssetTransaction( //AssetTransaction
+    public static Transaction assetAcceptanceTransaction( //AssetTransaction
```

Comment:
Since it's static, what do you think of naming it something like `CreateAssetAcceptanceTransaction`? same for the other onces

Reply:
How about this proposal: 
AssetCreateTransaction -> createAssetCreationTransaction or createAssetCreateTransaction 
assetConfigurationTransaction -> createAssetConfigurationTransaction or createAssetConfigureTransaction
assetAcceptanceTransaction -> createAssetAcceptanceTransaction or createAssetAcceptTransaction
assetDestroyTransaction -> createAssetDestroyTransaction
assetFreezeTransaction -> createAssetFreezeTransaction
assetRevokeTransaction -> createAssetRevokeTransaction
assetTransferTransaction -> createAssetTransferTransaction

Reply:
Well, the first ones are Objects - they should not get a verb. 
The last ones are static functions that return the object. I think that they should get the verb.